from_mwt_ds/DataScience/estimate.py

```diff
@@ -0,0 +1,15 @@
+import pandas as pd
+from estimators import bandits
+from estimators.bandits import ips
+
+def estimate_bucket(bucket: pd.DataFrame, config: dict) -> dict:
+    if len(bucket) == 0 or config == {}:
+        return {}
+    else:
+        e = bandits.ips.Estimator()
+        for row in bucket:
+            bucket.get(row)
```

Comment:
can you please describe what this line is doing exactly?

Reply:
```for every row in data frame```
I just wrote it exactly as you said to do. I can delete it as it's not really needed.

Reply:
I was asking more about 'bucket.get(row)' line. What have changed after this line is executed?